### DISCUSSION TITLE
Can Cebra be used for neuron clustering  like tsne instead of neuron dynamic analysis？

### DISCUSSION THREAD

**Discussion by Achuan-2** [AdaptiveMotorControlLab/CEBRA#162] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/351d0019-8f66-43f2-aa59-388937b212d6)

**Comment by MMathisLab** [comment]
Yes... we benchmark against it in our paper.

**Reply by Achuan-2** [reply]
However, after dimensionality reduction, the data usually forms a circular distribution, which is not conducive to clustering.

**Reply by MMathisLab** [reply, marked as answer]
well, no it's highly dependent on the data and what you are using as auxiliary variables; ie see Figure 2:

if you use only a discrete label (direction), you get discrete clusters:
<img width="495" alt="Screenshot 2024-07-24 at 14 28 55" src="https://github.com/user-attachments/assets/d4e9d162-9043-4560-b3e1-6d6e5ee73cab">

**Reply by Achuan-2** [reply]
Then, if I use cebra for unsupervised clustering, can't I obtain discrete clusters?
